packages/web/src/pages/upload-page/fields/CollectionTrackFieldArray.tsx

```diff
@@ -25,7 +25,7 @@ export const CollectionTrackFieldArray = () => {
                 {tracks.map((track, index) => (
                   <Draggable
                     key={track.file.name}
-                    draggableId={track.file.name}
+                    draggableId={track.file.name ?? ''}
```

Comment:
this is not a real world case, right? If we do actually render with `''` more than once in the list, it might break dragging behavior. But in theory we shouldn't get that far in the form with unnamed files.

Also curious how well our current logic handles drag n drop with multiple files of the same name

Reply:
feels like it shouldn't be possible to add two files with the same name 😬 

Changed to using `!` since we can safely assume there's a string name here. This is just an artifact of `NativeFile` again

Reply:
I'll have to test again. I think we were able to upload two with the same filename from different dirs, but maybe we handled that at the time I found it. I'll try it again tomorrow